docs/UserGuide.md

```diff
@@ -60,27 +65,56 @@ HealthSync is a **desktop app for managing patient details, optimised for use vi
 * Items in square brackets are optional.<br>
   e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
 
-* Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+* Items with `...` after them can be used multiple times including zero times.<br>
```

Comment:
Sorry, I'm not too sure what tags are. Is it just another field?

Reply:
Yes, it's just another field - it's a legacy field from AddressBook. I think we can include it in the optional fields section of our UG to update this commit.